lib/unit_tests.html

```diff
@@ -11,8 +11,8 @@
     <title>Nessie Javascript SDK Tests</title>
     
     <!-- QUnit Required Files -->
-    <link rel="stylesheet" href="//code.jquery.com/qunit/qunit-1.20.0.css">
-    <script src="//code.jquery.com/qunit/qunit-1.20.0.js"></script>
+    <link rel="stylesheet" href="http://code.jquery.com/qunit/qunit-1.20.0.css">
+    <script src="http://code.jquery.com/qunit/qunit-1.20.0.js"></script>
```

Comment:
no need for this. This is a relative reference. The url ```//code.jquery.com/qunit/qunit-1.20.0.js``` always for local server setup. Based on the current host, the browser automatically picks up the root http request. So that ```http``` isn't needed.